src/main/java/org/gnieh/logback/config/ConfigConfigurator.java

```diff
@@ -76,51 +77,32 @@ public void configure(LoggerContext loggerContext) {
 			} else {
 				addWarn("Invalid ROOT logger configuration. Ignoring it.");
 			}
-
 		}
 
 		Config loggerConfigs = config.getConfig("loggers");
 		for (Entry<String, ConfigValue> entry : loggerConfigs.root().entrySet()) {
 			if (entry.getValue() instanceof ConfigObject) {
-				configureLogger(loggerContext, appenders, entry.getKey(), loggerConfigs.getConfig(entry.getKey()),
-						false);
+				configureLogger(loggerContext, appenders, entry.getKey(), loggerConfigs.getConfig("\"" + entry.getKey() + "\""), false);
```

Comment:
Double-quotes required in `loggerConfigs.getConfig("\"" + entry.getKey() + "\"")` here and elsewhere to ensure the key name doesn't get interpreted as a path when it contains periods. This is specifically a problem here because the logger names reflect package names, and thus typically do contain periods.

Reply:
ok